packages/db-migrations/databases/fxa/patches/patch-138-139.sql

```diff
@@ -0,0 +1,25 @@
+-- Create carts table to store checkout Cart state when creating a subscription.
+-- An FxA user can have more than one Cart record e.g. if they have more than
+-- one subscription where each subscription was created in a different session.
+-- A Cart record may not be associated with an FxA user e.g. if they started
+-- the new user checkout flow but did not get to the point where we create an
+-- account.
+
+CALL assertPatchLevel('138');
+
+CREATE TABLE IF NOT EXISTS carts (
+    id BINARY(16) NOT NULL PRIMARY KEY,
+    uid BINARY(16),
+    state ENUM('start', 'processing', 'success', 'fail') NOT NULL,
```

Comment:
@bbangert is proposing to merge `'new'` and `'upgrade'` to a single `'start'` or similar state per [this comment in the Cart DB Structure doc](https://docs.google.com/document/d/1hUx-lbymI8It_Cn5AeUkVmwdc2_pqCJDqs8G9X6ia8c/edit?disco=AAAAz0TC_2k), so that's what I have here.

Reply:
We agreed to move forward with Ben's proposal. See meeting [notes](https://docs.google.com/document/d/1hUx-lbymI8It_Cn5AeUkVmwdc2_pqCJDqs8G9X6ia8c/edit#bookmark=id.xt28hdabrmhj) for more info.